crates/epaint/src/stroke.rs

```diff
@@ -52,3 +54,116 @@ impl std::hash::Hash for Stroke {
         color.hash(state);
     }
 }
+
+/// How paths will be colored.
+#[derive(Clone)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub enum ColorMode {
```

Comment:
Let's move this to its own file. In the future we may very well start using this for the fill color of primitives too (because why not!)

Reply:
did the thing in 41f819214bfac6bb9da35667ff4a91a0bf2cc5c6